doc/nrf/test_and_optimize/optimizing/power_nrf91.rst

```diff
@@ -1,23 +1,23 @@
-.. _app_power_opt_nRF9160:
+.. _app_power_opt_nRF91:
 
-Power optimization for nRF9160
-##############################
+Power optimization for nRF91 Series
+###################################
 
 .. contents::
    :local:
    :depth: 2
 
-In addition to the :ref:`general power optimization recommendations <app_power_opt_general>`, you can use the `Online Power Profiler (OPP)`_ when optimizing the power consumption of your nRF9160 application.
+In addition to the :ref:`general power optimization recommendations <app_power_opt_general>`, you can use the `Online Power Profiler (OPP)`_ when optimizing the power consumption of your cellular application.
 
-Online Power Profiler for LTE is a tool to estimate the current consumption of the nRF9160 LTE modem.
+Online Power Profiler for LTE is a tool to estimate the current consumption of the nRF91 Series LTE modem.
```

Comment:
This is not wrong, but a bit misleading: OPP only supports nRF9160 at the moment.